pkg/reconciler/apis/apibinding/index/names.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package index
+
+const (
+	// IndexAPIExportByIdentity is the indexer name for by identity index for the API Export indexers.
+	IndexAPIExportByIdentity = "byIdentity"
+)
```

Comment:
this is not our pattern and creates debt (a one-off we have to clean up afterwards). Please use https://github.com/kcp-dev/kcp/tree/main/pkg/indexers.

Reply:
up[dated